Exercises/arrays-methods (from gist)/arrays-methods (from gist).js

```diff
@@ -0,0 +1,100 @@
+//filter
+function allNonConsecutive (arr) {
+    return arr.filter((item, index) => index && arr[index - 1] !== item - 1).map((item, index) => ({i: arr.indexOf(item), n:item}));
+}
```

Comment:
You don't need `indexOf` because you have `index` parameter in your map function.